### DISCUSSION TITLE
台标最近都不显示

### DISCUSSION THREAD

**Discussion by ctkeji** [Guovin/iptv-api#762] [Ideas]
![874198c575618d3eb5443e0400d37ae](https://github.com/user-attachments/assets/c7772417-52f1-467c-954a-a8ec91be779d)
![874198c575618d3eb5443e0400d37ae](https://github.com/user-attachments/assets/564184f9-6859-4ff6-8851-7cea8668d368)

**Comment by Guovin** [comment]
台标.com域名在中国大陆无法访问，后续会更新使用新域名

**Comment by ctkeji** [comment]
谢谢。更改参数是在这里更改吗
![b84dc42fae1eab255780961b0bd57de](https://github.com/user-attachments/assets/4feb4d2d-c8b2-49ce-ba6f-440f625fc342)

**Reply by Guovin** [reply]
docker需要挂载config目录，修改config.ini文件

**Comment by Guovin** [comment]
已修复
